### PR TITLE
[doc/keras] incorrect comment in the example for `tf.keras.layers.Add` #27632

### PR DESCRIPTION
**Pull request by shashvatshahi1998**
Issue #27632

### CODE REVIEW

**Comment by shashvatshahi1998** [comment]
Removed typo

**Review by csukuangfj — COMMENTED** [review]
line 239 should be removed

**Comment by mihaimaruseac** [comment]
Please make sure the commits on the PR are only those needed for the PR. It makes integrating the code much harder to have commits touching other files.

You can create branches for each topic of work you chose to do, push the branches, make PRs with those branches only. Plus, on each branch you can `git squash`, `git rebase`, etc, to make everything look clean and neat and get approval and merging much faster.

File: tensorflow/python/keras/layers/pooling.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ class MaxPooling1D(Pooling1D):
   """Max pooling operation for temporal data.
 
   Arguments:
-    pool_size: Integer, size of the max pooling windows.
+    pool_size: Integer, size of the max  pooling windows.
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/layers/pooling.py]: This shouldn't be in this PR.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Please include only files that are needed for the PR. Use multiple branches, rebase them.

**Comment by shashvatshahi1998** [comment]
@mihaimaruseac I have removed that unwanted line but when I am trying to rebase my previous commits , I am facing some issues.

**Comment by mihaimaruseac** [comment]
Can you elaborate on the issues?

For future references, here's how I would do multiple PRs to the same repo:

1. fork the repo on github (delete old fork before)
2. `git clone` your fork
3. `git checkout -b $NAME` where `$NAME` would be the name of the branch
4. work only on one thing there, commit often
5. `git rebase -i ...` reorder commits, squash, cleanup
6. `git push`, make PR, go to step 3 with a different branch

Whenever you have to work on a PR you `git checkout` the corresponding branch, work on it and push and the PR updates automatically

**Comment by mihaimaruseac** [comment]
@shashvatshahi1998 ping on this?

**Comment by mihaimaruseac** [comment]
@shashvatshahi1998 should we also close this and reopen as a clean PR?

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**